src/Redux/mission/mission.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] @team it's not good practice to disable linters check for the entire file, kindly try and fix the error instead of disabling linters